src/formatter.test.ts

```diff
@@ -103,6 +105,25 @@ describe('Formatter', () => {
     stable(expected);
   });
 
+  it('comments', () => {
+    const source = `<!--
+    comment -->
+
+<!-- comment
+   with more
+  than one
+line  -->
+`;
+    const expected = `<!-- comment -->
+<!-- comment
```

Comment:
@rpaul-stripe do we want one newline after comments, or 2?

Reply:
I don't have a strong opinion on this.